File: week1/agree.c
#include <cs50.h>
#include <stdio.h>

// A program that prompts the user to agree or disagree.

int main(void)
{
    char c = get_char("Do you agree? ('y/Y' or 'n/N') ");

    if (c == 'y' || c == 'Y')
    {
        printf("Agreed.\n");
    }
    else if (c == 'n' || c == 'N')
    {
        printf("Not agreed.\n");
    }
    else
    {
        printf("Wrong answer.\n");
    }
}

File: week6/agree.py
# A program asking for agreement

from cs50 import get_string

# Ask user for agreement
s = get_string("Do you agree? ")

if s.lower() in ["y", "yes"]:
    print("Agreed.")
elif s.lower() in ["n", "no"]:
    print("Not agreed.")
else:
    print("Invalid input.")